modules/experimental-layers/src/gpu-grid-layer/gpu-grid-cell-layer-vertex.glsl.js

```diff
@@ -97,16 +97,11 @@ void main(void) {
   vec4 position_worldspace;
   gl_Position = project_position_to_clipspace(extrudedPosition, extrudedPosition64xyLow, offset, position_worldspace);
 
-  float lightWeight = 1.0;
-
-  if (extruded > 0.5) {
-    lightWeight = lighting_getLightWeight(
-      position_worldspace.xyz, // the w component is always 1.0
-      normals
-    );
+   if (extruded > 0.5) {
+    vec3 lightColor = lighting_getLightColor(color.rgb, project_uCameraPosition, position_worldspace.xyz, normals);
```

Comment:
I thought you were moving lighting to the fragment shaders?

Reply:
Based on discussion with Xiaoji, better to handle it in vertex shader for now for all the published layers, to avoid any performance issue, etc. Was trying to do it in fragment shader for mesh layer, but found there is no way to keep some layers in vertex shader while having extra in fragment shader.